src/service_broker/service_broker/nextflow.py

```diff
@@ -11,6 +11,11 @@ def build_nf_command(nf_script_path, workspace_dir):
     nf_command += f" --volume_dir {workspace_dir}"
     # nf_command += f" --workspace {workspace_dir}/"
     # nf_command += f" --mets {workspace_dir}/mets.xml"
+    # TODO: read user and pw from file or env. Leave blank skips storing in olahd
+    nf_command += " --olahd_username admin"
+    nf_command += ' --olahd_endpoint "http://141.5.104.244/api"'
+    nf_command += " --olahd_password JW24G.xR"
+    nf_command += " --nein oder"
```

Comment:
Why do we need this? I don't see the `--nein` parameter set or used anywhere in the Nextflow script.

Reply:
oh, sorry, that was a mistake, I wanted to test if not used parameters raise problems, and forgot to remove it again. Thanks for spotting I will remove that